CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
   - Rdma-core: `rdma-core-46.0-1`
   - Open MPI: `openmpi40-aws-4.1.5-1`
 - Upgrade Slurm to version 23.02.3.
+- Upgrade ARM PL to version 23.04.1 for Ubuntu 22.04 only.
```

Comment:
Why do we need to upgrade it only for Ubuntu22?

Reply:
It’s only necessary for ubuntu 22 since the older version is not available for ubuntu 22